test/powershell/Modules/Microsoft.PowerShell.Utility/FormatHex.Tests.ps1

```diff
@@ -5,219 +5,460 @@
-        ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        $actualResult = $result.ToString()
-        $actualResult | should Match "00000000   41 FF 7F FF FF FF 7F FF FF FF FF FF FF FF 7F     A"
-    }
+            $e | Should Not Be $null
+            $e.FullyQualifiedErrorId | Should Match "FileNotFound,Microsoft.PowerShell.Commands.FormatHex"
 
-    # handle bytes, int16, int32, and int64
-    It "Validate combined and with raw number formatting" {
-        $b = 65 # fits in a byte
-        $i16 = 32767 # fits in an int16
-        $i32 = 2147483647 # an int32
-        $i64 = 9223372036854775807 # an int64
-        # this will cause 2 lines to be emitted
-        $result = $b,$i16,$i32,$i64 | format-Hex -Raw
-        $result | Should Not BeNullOrEmpty
-        $result[0] | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        $result[1] | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        $result0 = $result[0].ToString()
-        $result0 | should match "00000000   41 00 00 00 FF 7F 00 00 FF FF FF 7F FF FF FF FF  A"
-        $result1 = $result[1].ToString()
-        $result1 | should match "00000010   FF FF FF 7F                                      .."
+            $a | Should Not Be $null
+            $a[0].ToString() | Should Match $inputText1
+        }
     }
 
-    # This test is to validate that streamed text does not have buffer underrun problems
-    It "ValidateEachBufferHasCorrectContentForStreamingText" {
-        $result = "a" * 30 | Format-Hex
-        $result | Should Not BeNullOrEmpty
-        ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
-        $actualResult = $result.ToString() -split "`r`n"
-        $actualResult.Count | should be 2
-        $actualResult[0].ToString() | Should be "00000000   61 61 61 61 61 61 61 61 61 61 61 61 61 61 61 61  aaaaaaaaaaaaaaaa"
-        $actualResult[1].ToString() | Should be "00000010   61 61 61 61 61 61 61 61 61 61 61 61 61 61        aaaaaaaaaaaaaa  "
-    }
+    Context "Cmdlet Functionality" {
+
+        It "Path is default Parameter Set 'fhx `$inputFile1'" {
 
-    # This test is to validate that files do not have buffer underrun problems
-    It "ValidateEachBufferHasCorrectContentForFiles" {
-        $result = Format-Hex -path $InputFile2
-        $result | Should Not BeNullOrEmpty
-        $result.Count | should be 2
-        $result[0].ToString() | Should be "00000000   54 68 69 73 20 69 73 20 61 20 62 69 74 20 6D 6F  This is a bit mo"
-        if ( $IsCoreCLR ) {
-            $result[1].ToString() | Should be "00000010   72 65 20 74 65 78 74                             re text         "
+            $result =  Format-Hex $inputFile1
+
+            $result | Should Not BeNullOrEmpty
+            ,$result | Should BeOfType 'Microsoft.PowerShell.Commands.ByteCollection'
+            $actualResult = $result.ToString()
+            ($actualResult -match $inputText1) | Should Be $true
```

Comment:
This should be '$actualResult | Should Match $inputText1'; otherwise, when the test fails, it going to say false not equals true.